### ISSUE TITLE
Which dataset used for training

### ISSUE THREAD

**Issue by chenyuntc** [abdo-eldesokey/nconv#6]
Hey there.
 I'm a little confused by which datasets you used for training `exp_guided_nconv_cnn_l2`, it seems to be `train+selval` according to [run-nconv-cnn.py#L67 ](https://github.com/abdo-eldesokey/nconv/blob/master/run-nconv-cnn.py#L67 ).
But it's validate on `selval` too. https://github.com/abdo-eldesokey/nconv/blob/122391fc4cd1ff51496f83099d9ba9fa3ff82a11/workspace/exp_guided_nconv_cnn_l2/error_selval_epoch_39.txt#L1

Also what's the difference between `nconv/workspace/exp_guided_nconv_cnn_l2/checkpoints/CNN_ep0039.pth.tar` and the one for test set, which datasets do you train it on.

**Comment by abdo-eldesokey** [comment]
Hi,
Both sets are fed to KittiDepthTrainer, however, the training only happens on the "training set" (see [KittiDepthTrainer.py#L113](https://github.com/abdo-eldesokey/nconv/blob/122391fc4cd1ff51496f83099d9ba9fa3ff82a11/trainers/KittiDepthTrainer.py#L113)

The checkpoint was trained on the training set and evaluated both on selval and the test set.
I hope this makes things clear.

**Comment by chenyuntc** [comment]
Got it

Thank You!
